reference/api/tasks.md

```diff
@@ -6,99 +6,98 @@ The `/tasks` route gives information about the progress of [asynchronous operati
 The task `uid` is incremented **globally.**
 :::
 
-## Get all tasks
+## Get tasks
 
 <RouteHighlighter method="GET" route="/tasks"/>
 
-List all tasks globally, regardless of index. The `task` objects are contained in the `results` array.
+List all tasks globally, regardless of index. The `task` objects are contained in the `results` array. Tasks are always returned in descending order of `uid`.
 
-### Example
+Task results are paginated and can be filtered.
 
-<CodeSamples id="get_all_tasks_1" />
+#### Query parameters
```

Comment:
That's a great point. They feel a little different from most parameters to me, but I'm sure that's bias because of how the feature was introduced to me. I'll add them here.

Curious if you agree @guimachiavelli ?